tendrl/commons/objects/node/atoms/check_required_services_running/__init__.py

```diff
@@ -0,0 +1,119 @@
+import etcd
+
+from tendrl.commons import objects
+from tendrl.commons.utils import etcd_utils
+from tendrl.commons.utils import log_utils as logger
+
+
+TENDRL_SERVICES = {
+    "server": [
+        "tendrl-node-agent",
+        "tendrl-monitoring-integration",
+        "tendrl-api",
+    ],
+    "storage_node": [
+        "tendrl-node-agent",
+        "glusterd",
+    ]
+}
+
+
+class CheckRequiredServicesRunning(objects.BaseAtom):
+    def __init__(self, *args, **kwargs):
+        super(CheckRequiredServicesRunning, self).__init__(*args, **kwargs)
+
+    def run(self):
+        node_context = NS.tendrl.objects.NodeContext().load()
+        tags = list(node_context.tags)
+        required_services_running = True
+        if "tendrl/monitor" in tags or \
+                "tendrl/integration/monitoring" in tags:
+            # check neccessary service status in server
+            required_services_running = self.check_service_status(
+                TENDRL_SERVICES["server"],
+                "server-node"
+            )
+        else:
+            # check neccessary service status in storage nodes
+            required_services_running = self.check_service_status(
+                TENDRL_SERVICES["storage_node"],
+                NS.node_context.fqdn
+            )
+            node_list = self.parameters['Node[]']
+            if required_services_running and len(node_list) > 1:
+                # check monitoring integration is running on server from
```

Comment:
What is this piece of code for? Why `tendrl-monitoring-integration` should be running on a storage node ?

Reply:
This atom is shared by un-manage and import flow, This is a pre-check to figure out whether all necessary services are running for the flow or not. 
For import flow job is processed by storage nodes, So I am checking node-agent, glusterd by executing ansible module and i am checking monitoring-integration status from ETCD.
Because setup alias is a monitoring integration job.  So before creating a job, the storage node will make sure monitoring-integration is running or not.

Reply:
Ack. Looks fine.